dot/network/message_test.go

```diff
@@ -5,125 +5,15 @@ package network
 
 import (
 	"encoding/hex"
-	"regexp"
 	"testing"
 
 	"github.com/ChainSafe/gossamer/dot/network/messages"
 	"github.com/ChainSafe/gossamer/dot/types"
 	"github.com/ChainSafe/gossamer/lib/common"
-	"github.com/ChainSafe/gossamer/lib/common/variadic"
 
 	"github.com/stretchr/testify/require"
 )
 
-func TestEncodeBlockRequestMessage(t *testing.T) {
```

Comment:
Should we not still be testing encoding here, just with new type?

Reply:
these tests uses a function called `NewUint32OrHashFromBytes` but this function is never used in production code, then I just removed it. What we use is the protobuf encoding mechanism, so we don't need to decode the parts of the message from bytes since `proto.Unmarshal` already gives us the correct message values